src/test/java/Homework18.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest {
+
+    @Test
+    public static void playSongTest() throws InterruptedException {
+        login("testguest@testpro.io","te$t$tudent");
+        playSongBtn();
+        playSong();
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/224788750-2e89f148-a307-474a-9e39-75a8272a8955.png)